packages/metrics/src/Metrics.ts

```diff
@@ -227,9 +227,9 @@ class Metrics implements MetricsInterface {
    * @decorator Class
    */
   public logMetrics(options: ExtraOptions = {}): HandlerMethodDecorator {
```

Comment:
Minor: could you please update also the occurrence at [L205](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/996dba61795b526cbfc0b209119b781b80e3c488/packages/metrics/src/Metrics.ts#L205), in the function comment? 

> A decorator automating coldstart capture, **throw** on empty metrics and publishing metrics on handler exit.